docs/molgenis/dev_apps.md

```diff
@@ -4,7 +4,9 @@ One of the features of MOLGENIS EMX2, is the ability to create your own frontend
 
 This guide will provide information on how to create a new app or how to contribute to an existing application. You will also find information on the tools that are used in MOLGENIS EMX2 and how to configure your app to deploy it in EMX2. In the last section (the appendices), you will find a step-by-step guides for developing apps.
 
-Before you begin, we encourage developers to read our [Development Guidelines](./dev_guidelines) to ensure consistency of all interfaces. If you notice any issue with this guide (e.g, broken example, outdated information, etc.), please open an issue on at [molgenis-emx2/issues](https://github.com/molgenis/molgenis-emx2/issues).
+Before you begin, we encourage developers to read our [Development Guidelines](./dev_guidelines) to ensure consistency of all interfaces.
+
+If you get stuck, have a look at the [Troubleshooting guide](./dev_troubleshooting). If you notice any issue with this guide (e.g, broken example, outdated information, etc.), please open an issue on at [molgenis-emx2/issues](https://github.com/molgenis/molgenis-emx2/issues).
```

Comment:
Do we need to link to the contributing here for the guidelines of opening issues (might be overkill here)?

Reply:
Good idea. I added a link to the contributing guide